osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -208,6 +209,27 @@ private void load(OsuConfigManager config, INotificationOverlay notificationOver
 
             // start all elements hidden
             hideTargets.ForEach(d => d.Hide());
+
+            logSkinLookupHierarchy(skinSource);
+        }
+
+        private void logSkinLookupHierarchy(ISkinSource skinSource)
```

Comment:
Sample output:

```csharp

[runtime] 2024-10-01 05:57:37 [verbose]: Skin lookup hierarchy for HUD:
[runtime] 2024-10-01 05:57:37 [verbose]: 
[runtime] 2024-10-01 05:57:37 [verbose]: | BeatmapSkinProvidingContainer
[runtime] 2024-10-01 05:57:37 [verbose]: -| SkinTransformer { Skin: LegacyBeatmapSkin { Name: Caravan Palace - Lone Digger (Ohai) [Bakari's Easy] } }
[runtime] 2024-10-01 05:57:37 [verbose]: -| RulesetSkinProvidingContainer
[runtime] 2024-10-01 05:57:37 [verbose]: --| SkinTransformer { Skin: ArgonSkin { Name: osu! "argon" (2022) (modified) (10) } }
[runtime] 2024-10-01 05:57:37 [verbose]: --| osu.Game.Skinning.ResourceStoreBackedSkin
[runtime] 2024-10-01 05:57:37 [verbose]: --| osu.Game.Skinning.SkinManager
[runtime] 2024-10-01 05:57:37 [verbose]: ---| ArgonSkin { Name: osu! "argon" (2022) (modified) (10) }
[runtime] 2024-10-01 05:57:37 [verbose]: ---| DefaultLegacySkin { Name: osu! "classic" (2013) }
[runtime] 2024-10-01 05:57:37 [verbose]: ---| ArgonSkin { Name: osu! "argon" (2022) }
[runtime] 2024-10-01 05:57:37 [verbose]: ---| TrianglesSkin { Name: osu! "triangles" (2017) }
```

Can remove or limit to debug if we decide this is too noisy.

Reply:
Make a quick change to fix the transformer lines:

```csharp
[runtime] 2024-10-01 09:05:57 [verbose]: Skin lookup hierarchy for HUD:
[runtime] 2024-10-01 09:05:57 [verbose]: 
[runtime] 2024-10-01 09:05:57 [verbose]: | BeatmapSkinProvidingContainer
[runtime] 2024-10-01 09:05:57 [verbose]: -| OsuLegacySkinTransformer { Skin: LegacyBeatmapSkin { Name: senzawa - i got blitzed and sang country road (mmi_) [C:LONJLANTRY RPJOASDS] } }
[runtime] 2024-10-01 09:05:57 [verbose]: -| RulesetSkinProvidingContainer
[runtime] 2024-10-01 09:05:57 [verbose]: --| OsuArgonSkinTransformer { Skin: ArgonSkin { Name: osu! "argon" (2022) (modified) (10) } }
[runtime] 2024-10-01 09:05:57 [verbose]: --| osu.Game.Skinning.ResourceStoreBackedSkin
[runtime] 2024-10-01 09:05:57 [verbose]: --| osu.Game.Skinning.SkinManager
[runtime] 2024-10-01 09:05:57 [verbose]: ---| ArgonSkin { Name: osu! "argon" (2022) (modified) (10) }
[runtime] 2024-10-01 09:05:57 [verbose]: ---| DefaultLegacySkin { Name: osu! "classic" (2013) }
[runtime] 2024-10-01 09:05:57 [verbose]: ---| ArgonSkin { Name: osu! "argon" (2022) }
[runtime] 2024-10-01 09:05:57 [verbose]: ---| TrianglesSkin { Name: osu! "triangles" (2017) }

```